# planner
v_max: 1.0
T_min: 0.5
T_max: 5.0
safe_dis: 0.6
delta_t: 0.1
weights: [1, 1, 1, 10000]
init_wpts_mode: fixed # 'fixed' or 'adaptive'
init_seg_len: 2.0
init_wpts_num: 2
init_T: 2.5
collision_cost_tol: 10
opt_tol: 1e-4

# mission
replan_mode: online # 'global', 'online', or 'periodic'
planning_time_ahead: 1.0
des_pos_z: 1.0
longitu_step_dis: 4.0
lateral_step_length: 1.0
target_reach_threshold: 0.2
cmd_hz: 60
selected_planner: nn # 'basic', 'batch', 'record', 'nn', 'neo', or 'warmstart'
replan_period: 1.0
record_metric: True
yaw_shift_tol: 0.17453